### DISCUSSION TITLE
Export to planner option?

### DISCUSSION THREAD

**Discussion by DeRpMeister** [lonqie/SchaleDB#70] [Ideas and Suggestions]
I just found out about the resource planner through this site after adding all of my characters.
Could there be a way to export into the planner format rather than the sites own format?

**Comment by lonqie** [comment]
Thanks for the suggestion.

Due to the differences in the data stored in exports, it would be a significant effort on my part to implement and _maintain through future updates_ an exporter that has compatibility with Justin's planner website, as it would need to provide default values for other data that Schale DB does not keep track of.

Would suggest forwarding on this idea to Justin himself and see if he wishes to enhance the import functionality to accept Schale DB collection export data. I think this would be much easier to implement on his end rather then attempting to go the other way around.
